### ISSUE TITLE
Have the bite mark/nommed section, not be on Heavy's lunchbox items after Heavy's eating animation.

### ISSUE THREAD

**Issue by TheShpeeCrab** [mastercomfig/team-comtress-2#460]
### Description

When the Heavy eats a lunchbox item, a bite mark/eaten section will appear on the lunchbox item. But the bite mark is still on the lunchbox item after the eating animation, so when the Heavy eats a lunchbox item again it will look like he is biting into nothing. I have some screenshots for examples of what the lunchbox item looks like after the eating animation. So having the bite mark/eaten section **not** be on the lunchbox item **after** the eating animation would then make it look like that the Heavy is biting into it instead of biting into nothing for the next time he eats the lunchbox item.
### Checklist

- [x] None of the open *or* closed issues document this idea.
- [x] This is a new idea, not a bug.
- [x] This idea does not seriously affect game balance.
- [x] This is obviously an improvement, not something that people commonly disagree on.
- [x] This is not an idea for new game content (e.g. new weapons, maps, sounds).

- [ ] This idea is for the code of the game, not for textures, sounds, or other resources.
- [ ] This idea has been implemented in vanilla TF2.
- [ ] There is proof that this idea is well liked (e.g. a history of use of a similar mod in a competitive league).
- [ ] This idea has been implemented in a mod. 
- [ ] This feature has existed in the vanilla game but was removed.
- [ ] This idea is for internal procedures or tooling (e.g. improving the build process, supporting new systems).

### Screenshots

![20201017084558_1](https://user-images.githubusercontent.com/72120470/96337920-715cb680-1058-11eb-8524-440c162d07ff.jpg)
![20201017085005_1](https://user-images.githubusercontent.com/72120470/96337922-73267a00-1058-11eb-9ad8-8ae852dde9f7.jpg)
![20201017085104_1](https://user-images.githubusercontent.com/72120470/96337923-74f03d80-1058-11eb-857e-2424e61a8f84.jpg)
![20201017085403_1](https://user-images.githubusercontent.com/72120470/96337925-76ba0100-1058-11eb-96fb-f7e93e6b923c.jpg)
![20201017085516_1](https://user-images.githubusercontent.com/72120470/96337926-7883c480-1058-11eb-925a-25dc2db469f5.jpg)
![20201017085529_1](https://user-images.githubusercontent.com/72120470/96337927-79b4f180-1058-11eb-8bc4-79e0f711ffed.jpg)

### Alternatives

1.
2. 
3.

**Comment by Melonenyoshi** [comment]
I think this is intended

**Comment by 107zxz** [comment]
he means when the sandvich recharges, it should not have a bite taken out of it. Its weird to see heavy bite the already missing part of the sandvich

**Comment by Melonenyoshi** [comment]
> he means when the sandvich recharges, it should not have a bite taken out of it. Its weird to see heavy bite the already missing part of the sandvich

I know but still

**Comment by agrastiOs** [comment]
@Melonenyoshi what suggests that this is intended?

**Comment by henriquecm** [comment]
@agrastiOs Why have a secondary model if it would only be visible a couple of frames?

**Comment by agrastiOs** [comment]
> 
> 
> @agrastiOs Why have a secondary model if it would only be visible a couple of frames?

It is definitely not a couple of frames. I feel the Sandvich's full model could be restored when the FOOD bar is full.

**Comment by henriquecm** [comment]
You can switch to sandvich/food items only when the bar is full, so the nommed model would only after the bite during the animation.
If you eat the sandvich at full health the bar remains full.
The scenario you want to keep the item nommed only last half of the eating animation.
I think this is just a minor nitpick.

**Comment by agrastiOs** [comment]
That's right, I'm just talking about a possible way to implement this code wise.
